packages/legend-graph/src/graphManager/protocol/pure/v1/model/context/V1_PureModelContextData.ts

```diff
@@ -18,9 +18,11 @@ import { V1_PureModelContext } from '../../model/context/V1_PureModelContext.js'
 import type { V1_PackageableElement } from '../../model/packageableElements/V1_PackageableElement.js';
 import type { V1_PureModelContextPointer } from '../../model/context/V1_PureModelContextPointer.js';
 import type { V1_Protocol } from '../../model/V1_Protocol.js';
+import type { Entity } from '@finos/legend-storage';
 
 export class V1_PureModelContextData extends V1_PureModelContext {
   origin?: V1_PureModelContextPointer | undefined;
   serializer?: V1_Protocol | undefined;
   elements: V1_PackageableElement[] = [];
+  INTERNAL_rawEntities: Entity[] = [];
```

Comment:
When will you remember my double underscore mate?

```suggestion
  INTERNAL__rawEntities: Entity[] = [];
```